virtual_ecosystem/models/hydrology/hydrology_tools.py

```diff
@@ -126,14 +127,15 @@ def setup_hydrology_input_current_timestep(
     # Select soil variables
     # FIXME - there's an implicit axis order built into these calculations (vertical
     #         profile is axis 0) that needs fixing.
+    # VIVI: I think this is implicit order is now baked in?
     output["top_soil_moisture_capacity_mm"] = (
```

Comment:
I think it is, we are not going to revert the order of layers?

Reply:
I can't see why we would want to! As long as these functions are always dealing with vertical layer x grid cells, then assuming the axis order seems easier and is probably faster.